Upload step for copy process completion added (#146)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processoperations-financeanalysis-basics.adoc

image:rpa_processOperations_FinanceAnalysis_Details_MonthlyFixedCosts_Example_Investments.png[rpa_processOperations_FinanceAnalysis_Details_MonthlyFixedCosts_Example_Investments]

> In our example, in May 2020, an initial investment of 120,000&nbsp;&euro; was made to buy the MuleSoft RPA solution and its environment. The depreciation period defined was 12&nbsp;months. This resulted in a monthly depreciation amount of 10,000&nbsp;&euro; for the next 12 months.

Reviewer: ```suggestion
In our example, in May 2020, an initial investment of 120,000&nbsp;&euro; was made to buy the MuleSoft RPA solution and its environment. The depreciation period defined was 12&nbsp;months. This resulted in a monthly depreciation amount of 10,000&nbsp;&euro for the next 12 months.
```

> In our example, in May 2020, an initial investment of 120,000&nbsp;&euro; was made to buy the MuleSoft RPA solution and its environment. The depreciation period defined was 12&nbsp;months. This resulted in a monthly depreciation amount of 10,000&nbsp;&euro; for the next 12 months.

Reviewer: ```suggestion
In this example, in May 2020, an initial investment of 120,000&nbsp;&euro; was made to buy the MuleSoft RPA solution and its environment. The depreciation period defined was 12&nbsp;months. This resulted in a monthly depreciation amount of 10,000&nbsp;&euro; for the next 12 months.
```

image:rpa_processOperations_FinanceAnalysis_Details_BreakEvenAnalysis_Example_Savings_ProcessRuns_ActualAndForecast.png[rpa_processOperations_FinanceAnalysis_Details_BreakEvenAnalysis_Example_Savings_ProcessRuns_ActualAndForecast]

In this example, which was also used in the section *Savings*, the process was already executed automatically in the months of November through April. Since the execution of the process is dependent on conditions that cannot be influenced - in this case, for example, fluctuations in the number of book orders - the process was executed a different number of times each month. Since the overall trend was increasing, the forecast of future execution frequency was set to a value that reflects this trend.

> If the forecast proves right, the savings grow with each month.

Reviewer: ```suggestion
If the forecast is accurate, the savings grow with each month.
```
